Cargo.toml

```diff
@@ -47,6 +47,7 @@ include = [
 [features]
 default = []
 schannel = []
+openssl3 = []
```

Comment:
Since openssl3 is a separate submodule, we need to update include paths above.

Reply:
This makes me think we need automation to validate all this stuff. I wonder if we should add a prereq step to our GitHub action that fake publishes the crate locally, then uploads it. Then all other jobs work off the crate, not a git clone.